code/RealMeAuthenticator.php

```diff
@@ -25,51 +25,26 @@ protected static function on_register()
         $cache = SS_Cache::factory('RealMeAuthenticator');
 
         $cacheKey = 'RegisterCheck';
-        if (true === (bool) $cache->load($cacheKey)) {
+        if ((bool) $cache->load($cacheKey)) {
```

Comment:
Nice catch, thanks. This is part of the original module. Have taken Jackson's changes for this one